app.js

```diff
@@ -0,0 +1,59 @@
+var express = require('express');
+var trucks = require('./trucks');
+
+var idManager = require('./idManager');
```

Comment:
Using this for the assignment means you have to change the links that client.js makes when appending new trucks to the list.